src/client/eppo-client.spec.ts

```diff
@@ -261,19 +265,20 @@ describe('EppoClient E2E test', () => {
   });
 
   describe('UFC Obfuscated Test Cases', () => {
-    const storage = new TestConfigurationStore();
-
     beforeAll(async () => {
-      mock.setup();
-      mock.get(flagEndpoint, (_req, res) => {
-        const ufc = readMockUFCResponse(OBFUSCATED_MOCK_UFC_RESPONSE_FILE);
-        return res.status(200).body(JSON.stringify(ufc));
-      });
+      global.fetch = jest.fn(() => {
+        return Promise.resolve({
+          ok: true,
+          status: 200,
+          json: () => Promise.resolve(readMockUFCResponse(OBFUSCATED_MOCK_UFC_RESPONSE_FILE)),
+        });
+      }) as jest.Mock;
+
       await init(storage);
     });
 
     afterAll(() => {
-      mock.teardown();
+      jest.restoreAllMocks();
```

Comment:
if we're only mocking fetch, we could just move this top the top-level `afterAll()`